lib/srv/db/cloud/watchers/watcher.go

```diff
@@ -126,7 +127,9 @@ func (w *Watcher) fetchAndSend() {
 			// others. This is acceptable, thus continue to other fetchers.
 			// DB agent may also query for resources that do not exist. This is ok.
 			// If the resource is created in the future, we will fetch it then.
-			if trace.IsAccessDenied(err) || trace.IsNotFound(err) {
+			// DB agent may be run into "unrecognized" engine filters in certain regions.
+			// This is also ok, other regions may be unaffected.
+			if trace.IsAccessDenied(err) || trace.IsNotFound(err) || common.IsUnrecognizedAWSEngineNameError(err) {
```

Comment:
doesn't rds and aurora fetcher capture the unrecognized error inside already?

Reply:
this is to cover the case where the describe engines and retry still fails with "unrecognized engine name" - I haven't seen that happen, but if it does I'd prefer we just skip the fetcher. I considered this case because this unrecognized error shouldn't have happened in the first place - AWS documents the engine name "aurora" as a valid engine name - so I'm just covering my bases here